client/src/main/java/org/apache/celeborn/client/ShuffleClientImpl.java

```diff
@@ -604,7 +608,6 @@ public void onSuccess(ByteBuffer response) {
                     .computeIfAbsent(shuffleId, (id) -> ConcurrentHashMap.newKeySet())
                     .add(mapKey);
               }
-              pushState.removeFuture(nextBatchId);
```

Comment:
why delete this line?

Reply:
it's my fault. i will commit another pr to add it back